src/modules/media/services/media.service.ts

```diff
@@ -167,6 +167,22 @@ export class MediaService {
 		return object;
 	}
 
+	public adaptLimitedMetadata(graphQlObject: GqlLimitedIeObject): Partial<Media> {
```

Comment:
this is not related to the limited vs extended metadata set, right?
https://meemoo.atlassian.net/browse/ARC-670